docs/en/02_Developer_Guides/08_Performance/02_HTTP_Cache_Headers.md

```diff
@@ -31,3 +31,23 @@ clause in `Cache-Control` and `Pragma` will be included.
 
 Used to set the modification date to something more recent than the default. [api:DataObject::__construct] calls 
 [api:HTTP::register_modification_date(] whenever a record comes from the database ensuring the newest date is present.
+
+### Vary: cache header
+
+By default, SilverStripe will output a `Vary` header (used by upstream caches for determining uniqueness) 
+that looks like
+
+```
+Cookie, X-Forwarded-Protocol, User-Agent, Accept
+```
+
+To change the value of the `Vary` header, you can change this value by specifying the header in configuration
+
+```yml
+HTTP:
+  vary: ""
```

Comment:
Does this actually work? I know that we've had issues with removing values in this way in the past.